### ISSUE TITLE
Simplify evaluator metadata

### ISSUE THREAD

**Issue by mattgarrish** [mattgarrish/epub-revision#2846]
### Section

https://www.w3.org/TR/epub-a11y-12/#sec-conf-reporting-eval

### Describe the problem

Considering the difficulty in extracting evaluator info for the metadata guide given the refines chains, does it still make sense to have so many refines chains going on?

For example, does it really make more sense that the evaluation date is attached to the evaluator name instead of directly to the conformance statement? Yes, the evaluator performed it on the specified date, but it's just as simple to say conformance was determined on the date. The same holds for the more detailed report.

And what do you do if you don't want to specify the evaluator, only the date. Then it has to bump up a level.

The only property that has to be attached to the evaluator is their credential.

### Describe the fix or new feature you propose

Drop the extra refines chain involving the evaluator and have these properties linked to the conformance statement.

**Comment by clapierre** [comment]
We recently had a Publisher in GCA add their own certifierReport adding it to the certifier refines vs. the conformsTo. I think I agree with you Matt I feel the data and additional certifierReport should be tied to the conformsTo metadata not specifically the certifiedBy and only the certifierCredential should be linked to the cetertifiedBy metadata.

**Comment by mattgarrish** [comment]
Ya, the only thing that gives me a bit of pause is that it means we'll have to add another xpath to the display guide variables for these, and can never really stop accounting for the double refines chain. Part of me says to just live with it.

**Comment by gregoriopellegrino** [comment]
I agree with simplifying the chain; in fact, in ONIX it is "flat," there is a date that applies to the entire record.

**Comment by mattgarrish** [comment]
I've been looking at this again and probably the simplest thing we could do is to make some of the existing uses of refines conditional on whether more than one conformance claim is made. I know at least one publisher adds their own statements, but I suspect that's not common practice.

So the default would be to not use refines unless adding more than one claim, in which case we keep the linking we have. The advantage of this approach over changing what refines what is that we don't have to do extensive changes to the display guide. It already accounts for this metadata not being attached to anything.

The only difficulty I'm running into with this is the evaluation date. Because we used a generic dcterms:date property attached to the evaluator, if we take refines away then it technically becomes a random date associated with the publication. Short of defining a11y:evaluationDate, all dc date properties are going to have the same problem.

But the display guide uses a dcterms:date without refines as the evaluation date even though it is error prone, so maybe we just live with it. I can't think of why a publisher would add that kind of generic date without attaching it to something, so the likelihood of it not being the evaluation date is probably low.
